### ISSUE TITLE
openFile but can't save it after change

### ISSUE THREAD

**Issue by syswombat** [keepassxreboot/keepassxc#7244]
## Overview
[TIP]:  # ( DO NOT include screenshots of your actual database! )
[NOTE]: # ( Give a BRIEF summary about your problem )
open a dkbx file change a password and can't safe the file

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. a file have been crated with keepassX on a Old debian Linux laptop
2.  new laptop with Debian and keepassXC
3.  copy the kdbx file from the old to the new laptop
4. open the file on the new laptop and use the file - no Problem
5. change a password
6. can't save the file

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
to be able to save the file

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )
can't save the file

## Context
[NOTE]: # ( Give us any additional information you may have. )

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - 2.6.2
Revision: REVISION

[NOTE]: # ( Pick choices based on your environment )
Operating System: Windows/Linux/macOS
Desktop Env: Gnome/KDE/XFCE/Mate/Cinnamon
Windowing System: X11/Wayland

**Comment by droidmonkey** [comment]
What is the error message displayed when you try to save it? Did you try to get fancy and setup special file system permissions on the file? Also recommend updating to 2.6.6.

**Comment by syswombat** [comment]
no Error, the Disk is greyed out so i can't save!

and to save a normal file works fine

**Comment by droidmonkey** [comment]
That means auto save is enabled and the file is being saved every time you make a change. You can disable that in the application settings.

**Comment by syswombat** [comment]
Ok, yes your are correct - so it is solved - MERCI

**Comment by LzGnius** [comment]
Goodmoring, I have a simil problem. I have a .kdbx file saved on Google drive linked to the file system of my laptop (Ubuntu). I can open the db and edit it, but after I close it every changes aren't saved. I noticed that there is auto-save enabled but didn't work. I tried to disable any type of automatic savings, trying to save every changes manually before quit. And this another issue: "Save database" funcion is greyed, and i can't save manually.
On my Windows laptop I have the same default settings with auto-save, and it works perfec.
Thanks in advice for the support.
